src/main/steps/applicant1/hub-page/joint/form.njk

```diff
@@ -11,17 +11,16 @@
       text: confirmReceipt,
       classes: form.submit.classes,
       type: "submit",
-      name: "applicant1ConfirmReceipt",
+      name: applicantConfirmReceipt,
```

Comment:
you changed these but are they not used when posting to the backend, do we not need to change it there?

Reply:
I pass the correct string through content

Reply:
`const applicantConfirmReceipt = isApplicant2 ? 'applicant2ConfirmReceipt' : 'applicant1ConfirmReceipt';
  const applicantApplyForConditionalOrderStarted = isApplicant2
    ? 'applicant2ApplyForConditionalOrderStarted'
    : 'applicant1ApplyForConditionalOrderStarted';`